lib/nylas/api.rb

```diff
@@ -99,6 +99,13 @@ def revoke(access_token)
       response.code == 200 && response.empty?
     end
 
+    # Returns list of IP addresses
+    # @return [Boolean]
```

Comment:
What are you expecting to be returned here? This says both a boolean and a list of ip addresses

Reply:
If you look above at this `@labels ||= Collection.new(model: Label, api: self)` type situation a few lines above, I think that's basically what you're going to want to do here. So you'll have to start with making an `IPAddress` model. Maybe see if you feel comfortable trying to follow that path and then lmk if you want to pair.

Reply:
Sounds good - I'll work on this tomorrow and ping you if I get stuck.